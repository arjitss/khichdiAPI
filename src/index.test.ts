import a from './index';

/**
 * This is an example test function.
 *
 * @remarks
 * This test method covers the methods written in index.ts.
 *
 */

describe('Test Framework', () => {
  test('Sample Test', () => {
    // Arrange
    const x = 2;
    const y = 3;
    // Act
    const z = a(x, y);
    // Assert
    expect(z).toBe(5);
    expect(z).toMatchSnapshot();
  });
});
