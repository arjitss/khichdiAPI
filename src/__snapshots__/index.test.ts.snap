// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Framework Sample Test 1`] = `5`;
